src/sentry/api/bases/organization_events.py

```diff
@@ -57,14 +60,6 @@ def get_snuba_query_args(self, request, organization, params):
                 snuba_args, reference_event_id
             )
 
-        # TODO(lb): remove once boolean search is fully functional
-        has_boolean_op_flag = features.has(
-            "organizations:boolean-search", organization, actor=request.user
-        )
-        if snuba_args.pop("has_boolean_terms", False) and not has_boolean_op_flag:
-            raise OrganizationEventsError(
-                "Boolean search operator OR and AND not allowed in this search."
-            )
         return snuba_args
 
     def get_snuba_query_args_legacy(self, request, organization):
```

Comment:
Can we get rid of get_snuba_query_args altogether ?

Reply:
I would like to come back and address this file once this gets merged, as it might require some larger restructuring.